lib/chewy/search.rb

```diff
@@ -17,8 +16,7 @@ module Chewy
   # inherited from {Chewy::Search::Request}
   #
   # Also, the search class is refined with one of the pagination-
-  # providing modules: {Chewy::Search::Pagination::Kaminari} or
-  # {Chewy::Search::Pagination::WillPaginate}.
+  # providing modules: {Chewy::Search::Pagination::Kaminari}
```

Comment:
The suggestion feature is not available on deleted line, but I suggest changing this paragraph to:

```
Also, the search class is refined with the pagination module {Chewy::Search::Pagination::Kaminari}.
```